core/src/message.rs

```diff
@@ -92,6 +92,25 @@ pub struct UnregisterUser {
     pub user_id: Id,
 }
 
+/// Register a member of an org on the Radicle Registry with the given ID.
+///
+/// # State-dependent validations
+///
+/// The involved org must exit.
```

Comment:
Alexis uses "identified" which I quite like:

```suggestion
/// The identified org must exit.
```